types/autocannon/autocannon-tests.ts

```diff
@@ -0,0 +1,68 @@
+import * as autocannon from 'autocannon';
```

Comment:
`autocannon` is being imported as namespace and they are not callable. Instead use `import autocannon from 'autocannon'`;

Reply:
It seems I can call the namespace without any compiling problem, but let's make the change :)
(Default import does not work without module interop option)